public/vendor/formatjs/integrations.js

```diff
@@ -6,3 +6,9 @@ window.HandlebarsIntl = HandlebarsIntl;
 
 import ReactIntl from 'react-intl';
 window.ReactIntl = ReactIntl;
+
+import IntlRelativeFormat from 'intl-relativeformat';
+window.IntlRelativeFormat = IntlRelativeFormat;
+
+import IntlMessageFormat from 'intl-messageformat';
+window.IntlMessageFormat = IntlMessageFormat;
```

Comment:
Can we add an __addLocaleData() function onto Ember Intl in some way?

ember-intl currently depends on the globals IntlMessageFormat and IntlRelativeFormat.

> Ember Intl supports being imported as an ES6 module like the rest of the integration libs, right?

It doesn't